### ISSUE TITLE
Trial parallelism on a slurm scheduler?

### ISSUE THREAD

**Issue by hbaldwin07** [keras-team/keras-tuner#383]
I am training keras models via an HPC resource which uses SLURM scheduler. I cannot figure out how to get the trial parallelism to work as described in https://keras-team.github.io/keras-tuner/tutorials/distributed-tuning/. The tensorflow mirrored strategy (in tuner.search) for data parallelism seems to work fine. I've tried troubleshooting with the HPC staff, but am still stumped.

Does anyone have any advice/instructions on how to use the built-in distributed tuning strategy of keras-tuner on a SLURM cluster? 
Tensorflow has a tf.distribute.cluster_resolver.SlurmClusterResolver function, but I'm not sure how I would use that for keras-tuner? Is there a way to setup my own parallel tuners and just bypass the built-in oracle-chief / worker scripts?

(Sorry for any naïveté, I'm relatively new at tensorflow and python. I've tried my best to search all documentation, tutorials, etc but I am totally stumped)
